WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/ShippingLabelFormViewModel.swift

```diff
@@ -283,18 +283,32 @@ final class ShippingLabelFormViewModel {
     }
 
     /// Returns the body of the Package Details cell
-    /// TODO-4599: Update this for multi-package case.
     ///
     func getPackageDetailsBody() -> String {
-        guard let packagesResponse = packagesResponse,
-              let selectedPackage = selectedPackagesDetails.first else {
+        guard selectedPackagesDetails.isNotEmpty else {
             return Localization.packageDetailsPlaceholder
         }
 
-        let packageTitle = searchCustomPackage(id: selectedPackage.packageID)?.title ?? searchPredefinedPackage(id: selectedPackage.packageID)?.title ?? ""
+        let packageTitle: String
```

Comment:
nit: Since this string is the package title when there's 1 package, or a count of the items and packages when there are multiple packages, maybe it could be named something more generic e.g. `packageDescription` instead of `packageTitle`.